alacritty_terminal/src/term/mod.rs

```diff
@@ -801,7 +722,14 @@ impl SizeInfo {
 }
 
 pub struct Term<T> {
-    /// Terminal focus.
+    /// Terminal requires redraw.
+    pub dirty: bool,
+
+    /// Vi mode cursor.
+    pub vi_mode_cursor: ViModeCursor,
+
+    pub selection: Option<Selection>,
+    pub visual_bell: VisualBell,
     pub is_focused: bool,
```

Comment:
I removed them because I didn't necessarily see a point in describing something with just its name.

It might be worth thinking about the best way to do this in general, maybe @nixpulvis has an opinion too.

But basically commenting `selection` on terminal with `/// Terminal selection.` doesn't really serve any purpose.